dev-java/ant_core/ant_core-1.10.7.recipe

```diff
@@ -20,9 +20,9 @@ The Apache Ant project is part of the Apache Software Foundation.
 HOMEPAGE="http://ant.apache.org/"
 COPYRIGHT="1999-2018 The Apache Software Foundation"
 LICENSE="Apache v2"
-REVISION="1"
+REVISION="2"
```

Comment:
No need to bump revision here (it's not merged yet) :)

Reply:
Yeah, I just fixed it.